app/main.py

```diff
@@ -1,3 +1,81 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: None) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(km=self.km + other)
+        else:
+            raise TypeError(f"unsupported operand type(s) for +: "
+                            f"Distance and {type(other)}")
+
+    def __iadd__(self, other: None) -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += other
+        else:
+            raise TypeError(f"unsupported operand type(s) for +: "
+                            f"Distance and {type(other)}")
+        return self
+
+    def __mul__(self, koef: int) -> "Distance":
+        return Distance(km=self.km * koef)
+
+    def __truediv__(self, num: int) -> "Distance":
+        if num == 0:
+            raise ZeroDivisionError("You can't divide by zero!")
+        return Distance(round(self.km / num, 2))
+
+    def __lt__(self, other: None) -> bool:
```

Comment:
The type annotation for the `other` parameter should not be `None`. It should be `Union[Distance, int, float]` to indicate that it can accept a `Distance` object or a numeric type.